a10_octavia/controller/worker/tasks/nat_pool_tasks.py

```diff
@@ -0,0 +1,116 @@
+#    Copyright 2020, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from oslo_log import log as logging
+from taskflow import task
+
+import acos_client.errors as acos_errors
+
+from a10_octavia.controller.worker.tasks.decorators import axapi_client_decorator
+
+LOG = logging.getLogger(__name__)
+
+
+class NatPoolCreate(task.Task):
```

Comment:
This task needs to also have a revert method

Reply:
Added revert method for this